### DISCUSSION TITLE
how can i tell what version I of Open MCT I am currently using?

### DISCUSSION THREAD

**Discussion by rhoran1961** [nasa/openmct#6069] [Community Q&A] [answered]
I downloaded open mct for a project early in the year. Is there a way to tell what version of open mct I am currently using?

**Comment by marcelo-earth** [comment, marked as answer]
Hi @rhoran1961!

In the top-right of the platform you can see the Open MCT logo. Once you click on it, it will open a modal with information about Open MCT, and there is a **Version Information** headline with the version tag.

<img width="445" alt="Open MCT logo at the top right corner" src="https://user-images.githubusercontent.com/47821093/217152270-70f250c5-b2c6-45e0-a769-0829ae96426e.png">

<img width="1437" alt="Open MCT information" src="https://user-images.githubusercontent.com/47821093/217152463-1a686375-708a-41c9-92b7-480c313123b3.png">
